integration/testrunner/grpcrunner/grpc.go

```diff
@@ -252,13 +292,21 @@ func createConnection(tb testing.TB, addr string, idToken string) *grpc.ClientCo
 // This query is used to find the relevant audit log in BigQuery, which we assume will be added by the server.
 // We specifically look up the log using the UUID specified in the request as we know the server will add that
 // as the resource name, and provides us a unique key to find logs with.
-func (g *GRPC) makeQueryForGRPCUnary(u uuid.UUID) *bigquery.Query {
-	queryString := fmt.Sprintf("SELECT count(*) FROM %s.%s WHERE jsonPayload.resource_name=? LIMIT 1", g.ProjectID, g.DatasetQuery)
+func (g *GRPC) makeQueryForGRPCUnary(u uuid.UUID, justificationRequired bool) *bigquery.Query {
+	var additionalConditionals string
+	if justificationRequired {
```

Comment:
It might be better to build the string as we go:

```go
q := fmt.Sprintf("SELECT count(*) FROM %s.%s WHERE jsonPayload.resource_name = ?", g.ProjectID, g.DatasetQuery)

if justificationRequest {
  q += ` AND jsonPayload.metadata.justification_token != ""`
}

q += ` LIMIT 1`
return utils.MakeQuery
```